licenses/eclipse_public_license_1.0.html

```diff
@@ -14,7 +14,7 @@
     <meta name="keywords" content="edl, legal, license"/>
     <link rel="shortcut icon" href="/eclipse.org-common/themes/solstice/public/images/favicon.ico"/>
     <title>Eclipse Distribution License | The Eclipse Foundation</title>
-    <link rel="stylesheet" href="/eclipse.org-common/themes/solstice/public/stylesheets/quicksilver.min.css?v0.105"/>
+    <link rel="stylesheet" href="/eclipse.org-common/themes/solstice/public/stylesheets/quicksilver.min.css?v0.127"/>
```

Comment:
remove Lincenses changes in this PR, this file and licenses/cddl_gplv2_ce.html

Reply:
Removed the changes for these licenses.